CHANGES.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - We no longer pass the `epoch` parameter to LR schedulers, since that parameter has been deprecated. We now rely on the scheduler to keep track of the epoch itself.
+- Changed implementation of `net.history` access to make it faster; this should not have any noticeable effects, but if you encounter bugs, please create an issue
```

Comment:
```suggestion
- Changed implementation of `net.history` access to make it faster, which should be most noticeable for small models/datasets.
```

Reply:
Yes, thanks, for sure this entry should mention what the benefit is. I included it but didn't use exactly your proposal.